src/main/java/org/embulk/input/jira/util/JiraUtil.java

```diff
@@ -0,0 +1,271 @@
+package org.embulk.input.jira.util;
+
+import com.google.gson.JsonElement;
+
+import org.embulk.config.ConfigException;
+import org.embulk.input.jira.Issue;
+import org.embulk.input.jira.JiraInputPlugin.PluginTask;
+import org.embulk.spi.Column;
+import org.embulk.spi.ColumnConfig;
+import org.embulk.spi.ColumnVisitor;
+import org.embulk.spi.PageBuilder;
+import org.embulk.spi.Schema;
+import org.embulk.spi.json.JsonParser;
+import org.embulk.spi.time.Timestamp;
+import org.embulk.spi.time.TimestampParser;
+
+import javax.ws.rs.core.UriBuilder;
+
+import java.io.IOException;
+import java.net.HttpURLConnection;
+import java.net.URL;
+import java.util.List;
+import java.util.stream.Collectors;
+import java.util.stream.StreamSupport;
+
+import static com.google.common.base.Strings.isNullOrEmpty;
+import static org.embulk.input.jira.Constant.DEFAULT_TIMESTAMP_PATTERN;
+
+public final class JiraUtil
+{
+    private JiraUtil() {}
+
+    public static int calculateTotalPage(int totalCount, int resultPerPage)
+    {
+        return (int) Math.ceil((double) totalCount / resultPerPage);
+    }
+
+    public static String buildPermissionUrl(String url)
+    {
+        return UriBuilder.fromUri(url)
+                .path("rest")
+                .path("api")
+                .path("latest")
+                .path("myself").build().toString();
+    }
+
+    public static String buildSearchUrl(String url)
+    {
+        return UriBuilder.fromUri(url).
```

Comment:
Nitpick: the trailing dot should be on the next line. Purely my taste, not necessary to change: the builder style for multiple segments benefits when it is mixed with variables; for constants, `...fromUri(url).path("rest/api/latest/myself")...` looks a bit more concise and still easy to read.

Reply:
Done